docs/deployment/integrations/kuberay.md

```diff
@@ -0,0 +1,29 @@
+---
+title: KubeRay
+---
+[](){ #deployment-kuberay }
+
+[KubeRay](https://github.com/ray-project/kuberay) provides a Kubernetes-native way to run vLLM workloads on Ray clusters.
+A Ray cluster can be declared in YAML, and the operator then handles pod scheduling, networking configuration, restarts, and rolling upgrades—all while preserving the familiar Kubernetes experience.
+
+---
+
+## Why KubeRay instead of manual scripts?
+
+| Feature | Manual scripts | KubeRay |
+|---------|-----------------------------------------------------------|---------|
+| Cluster bootstrap | Manually SSH into every node and run a script | One command to create or update the whole cluster: `kubectl apply -f cluster.yaml` |
+| Fault-tolerance | Nodes must be restarted by hand | Pods are automatically rescheduled; head-node fail-over supported |
+| Autoscaling | Unsupported | Native horizontal **and** vertical autoscaling via Ray Autoscaler & Kubernetes HPA |
+| Upgrades | Tear down & re-create manually | Rolling updates handled by the operator |
+| Monitoring | ad-hoc | Distributed observability with Ray Dashboard |
+| Declarative config | Bash flags & environment variables | Git-ops-friendly YAML CRDs (RayCluster/RayService) |
+
+Using KubeRay reduces the operational burden and simplifies integration of Ray + vLLM with existing Kubernetes workflows (CI/CD, secrets, storage classes, etc.).
```

Comment:
Feedback:

KubeRay doesn't support "update" RayCluster.
Pod restart behavior is based on the Pod's restartPolicy set by user.
Ray doesn't support HPA and scale vertically (at least for now).
We don't support rolling upgrade. We use blue/green upgrade for now.
Users can still use Ray dashboard if they manually launch Ray cluster.

Reply:
Thanks! How's this: 

- One command to create ~~or update~~ ...
- ~~Pods are automatically rescheduled; head-node fail-over supported~~ --> Pod restarts supported, including for head node.
- ~~| Autoscaling | Unsupported | Native horizontal **and** vertical autoscaling via Ray Autoscaler & Kubernetes HPA |~~
- ~~Rolling updates handled by the operator~~ --> Blue/green deployment updates supported 
- ~~ad-hoc~~ --> Manual port-forwarding required..? I suppose that's also true of k8s though